flake/dev/generate-all-maintainers/generate-all-maintainers.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p python3
+"""
+Generate all-maintainers.nix using meta.maintainers as source of truth.
+
+This script uses the meta.maintainers system to extract maintainer information
+by evaluating an empty nixvimConfiguration, which is much simpler and more
+reliable than parsing files with regex.
+"""
+
+import argparse
+import json
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict
+
+
+class MetaMaintainerGenerator:
+    """Generates maintainers list using meta.maintainers from nixvim evaluation."""
+
+    def __init__(self, nixvim_root: Path):
+        self.nixvim_root = nixvim_root
+        self.nixvim_maintainers_file = nixvim_root / "lib" / "maintainers.nix"
+        self.output_file = nixvim_root / "all-maintainers.nix"
+        self.extractor_script = nixvim_root / "flake" / "dev" / "generate-all-maintainers" / "extract-maintainers-meta.nix"
+
+    def extract_maintainers_from_meta(self) -> Dict:
+        """Extract maintainer information using meta.maintainers."""
+        print("🔍 Extracting maintainers using meta.maintainers...")
+
+        try:
+            result = subprocess.run(
+                ["nix", "eval", "--impure", "--file", str(self.extractor_script), "--json"],
+                capture_output=True,
+                text=True,
+                timeout=60,
+            )
+
+            if result.returncode == 0:
+                data = json.loads(result.stdout)
+                print("✅ Successfully extracted maintainers using meta.maintainers")
+                return data
+            else:
+                print(f"❌ Failed to extract maintainers: {result.stderr}")
+                sys.exit(1)
+
+        except subprocess.TimeoutExpired:
+            print("❌ Timeout while extracting maintainers")
+            sys.exit(1)
+        except Exception as e:
+            print(f"❌ Error extracting maintainers: {e}")
+            sys.exit(1)
+
+    def format_maintainer_entry(self, name: str, info: Dict, source: str) -> str:
+        """Format a single maintainer entry with nix fmt compatible formatting."""
```

Comment:
It's frustrating that we need to output as nix. Ideally the rfc39 impl would also support importing JSON. cc @zowoq 

If you want to avoid all this boilerplate, you could use `lib.generators.toPretty`, although that's not guaranteed to be nixfmt compliant, it _should_ be.

Still, JSON is the "correct" way to store such static/generated data...